backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidApplication.java

```diff
@@ -214,18 +192,14 @@ protected void createWakeLock (boolean use) {
 		}
 	}
 
+	@TargetApi(Build.VERSION_CODES.HONEYCOMB)
 	protected void hideStatusBar (boolean hide) {
 		if (!hide || getVersion() < 11) return;
 
 		View rootView = getWindow().getDecorView();
 
-		try {
-			Method m = View.class.getMethod("setSystemUiVisibility", int.class);
-			if (getVersion() <= 13) m.invoke(rootView, 0x0);
-			m.invoke(rootView, 0x1);
-		} catch (Exception e) {
-			log("AndroidApplication", "Can't hide status bar", e);
-		}
+		if (getVersion() <= 13) rootView.setSystemUiVisibility(0x0);
```

Comment:
Does anybody know why it is this way?
Isn't the uiVisibility always changed to 0x1 the line below?

Reply:
No idea, I think many things could be cleaned up because the whole immersive mode / status bar code is a bit of a mess right now. Have a look at https://github.com/libgdx/libgdx/issues/3335 and https://github.com/libgdx/libgdx/pull/5329.